runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/servertype/impl/CRCServerType.java

```diff
@@ -14,4 +17,19 @@ public CRCServerType(String id, String name, String desc) {
 	public IServerDelegate createServerDelegate(IServer server) {
 		return new CRCServerDelegate(server);
 	}
+	
+	@Override
+	protected void fillRequiredAttributes(CreateServerAttributesUtility attrs) {
```

Comment:
Overriding these two methods is how we designate required and optional attributes.